internal/template/templates/.claude/skills/moai-domain-uiux/modules/design-system-tokens.md

```
@@ -0,0 +1,441 @@
+description: Moai Design Systems - Professional implementation guide
+version: 1.0.0
+modularized: false
+tags:
+ - enterprise
+ - systems
+ - development
+updated: 2025-11-24
+status: active
+---
+
+## Quick Reference (30 seconds)
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)